code/datums/wires/_wires.dm

```diff
@@ -252,8 +252,8 @@
 	else if(user.is_holding_item_of_type(/obj/item/multitool/abductor))
 		reveal_wires = TRUE
 
-	// Station blueprints do that too, but only if the wires are not randomized.
-	else if(user.is_holding_item_of_type(/obj/item/areaeditor/blueprints) && !randomize)
```

Comment:
CE blueprints now tell you the wires of syndicate bombs

Reply:
Sounds like a good functionality to cut to me then. 
As already discussed, it's kinda moot for them to display this since engineers have the right tools as it is and this isn't a function that greatly assists antagonists either.